easy_my_coop_website/views/subscription_template.xml

```diff
@@ -236,6 +236,26 @@
     </template>
 
     <template id="rules_template" name="rules_template">
+
+
+        <div
+            t-if="res_company.allow_id_card_upload"
+            t-attf-class="form-group #{error and 'file' in error and 'has-error' or ''}"
+            name="resume_container"
+        >
+        <label for="Resume">Idendity
+            card scan
+        </label>
+        <div class="col-md-7 col-sm-8">
+            <input
+                    type="file"
+                    class="form-control"
+                    name="Resume"
+                    t-attf-value="#{file or ''}"
+                />
+        </div>
+    </div>
```

Comment:
1. Are we sure that `Resume` is for identity cards, and not for CVs?
2. Should `Resume` be lower-case instead?

Reply:
Ah yes I think it has been copy pasted from another module to upload resumes ! ^^ I'll fix it, thx !

Reply:
Actually I prefer not doing this because there is risk of bug. I don't where this field is used in the code and we since we don't have time for another round of test I think keeping this as it is a better option.

Reply:
Can you open an issue for this? So we can look at it some other time.